lib/fizzy/parser_expr.cpp

```diff
@@ -88,50 +83,55 @@ parser_result<std::optional<ValType>> parse_blocktype(const uint8_t* pos, const
     return {validate_valtype(type), pos};
 }
 
-void update_stack_height(ControlFrame& frame, int stack_height_required, int stack_height_change)
+void update_operand_stack(const ControlFrame& frame, Stack<ValType>& operand_stack,
+    span<const ValType> inputs, span<const ValType> outputs)
```

Comment:
```suggestion
    const InstructionType& type)
```

Reply:
This won't work, because this function is used for both `InstructionType` and `FuncType`, that's the whole reason of using `span` here.